joern-cli/frontends/kotlin2cpg/src/main/scala/io/joern/kotlin2cpg/types/DefaultTypeInfoProvider.scala

```diff
@@ -743,6 +754,13 @@ class DefaultTypeInfoProvider(environment: KotlinCoreEnvironment) extends TypeIn
       }
       .getOrElse(defaultValue)
   }
+  def typeFromImports(name: String, file: KtFile): Option[String] = {
+    file.getImportList.getImports.asScala.flatMap { directive =>
+      if (directive.getImportedName != null && directive.getImportedName.toString == name.stripSuffix("?"))
+        Some(directive.getImportPath.getPathStr)
+      else None
```

Comment:
The `if (condition) { Some(thing) } else { None }` pattern can also be rewritten as `Option.when(condition) { thing }`

Reply:
same as above